lib/usd/ui/views/USDImportDialog.cpp

```diff
@@ -0,0 +1,245 @@
+//
+// Copyright 2019 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "USDImportDialog.h"
+#include "factories/TreeModelFactory.h"
+//#include "TreeHeader.h"
+#include "ItemDelegate.h"
+#include "ui_USDImportDialog.h"
+
+#include <stdexcept>
+
+MAYAUSD_NS_DEF {
+
+IUSDImportView::~IUSDImportView() { }
```

Comment:
Why put this here? Seems like it should just be declared `= default` in the header.